src/sentry/godot_error_types.h

```diff
@@ -11,25 +10,25 @@ namespace sentry {
 // Godot Engine error types that we can detect.
 // As defined in: https://github.com/godotengine/godot/blob/04692d83cb8f61002f18ea1d954df8c558ee84f7/core/io/logger.h#L51-L56
 enum class GodotErrorType {
-	ERROR_TYPE_ERROR,
-	ERROR_TYPE_WARNING,
-	ERROR_TYPE_SCRIPT,
-	ERROR_TYPE_SHADER,
+	ERROR_TYPE_ERROR = 0,
+	ERROR_TYPE_WARNING = 1,
+	ERROR_TYPE_SCRIPT = 2,
+	ERROR_TYPE_SHADER = 3,
 };
 
 // Enum used with bitwise operations to represent the set of Godot error types that the Sentry logger should capture.
 enum class GodotErrorMask {
-	NONE = 0,
-	ERROR = 1 << int(GodotErrorType::ERROR_TYPE_ERROR),
```

Comment:
MSVC is tripping on ERROR for no apparent reason - compiler error messages were not helpful. So I renamed these constants.